logging/src/main/java/alluxio/logging/scope/ScopeWorkerFlowLoggingAspect.java

```diff
@@ -0,0 +1,42 @@
+package alluxio.logging.scope;
+
+
+import alluxio.logging.scope.base.ScopeFlowLoggingAspect;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+
+/**
+ * This class represents a {@link alluxio.logging.base.FlowLoggingAspect} for the
+ * Alluxio Worker, logging and keeping track of all method calls it executes at any
+ * given moment.
+ */
+@Aspect
+public final class ScopeWorkerFlowLoggingAspect extends ScopeFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "ScopeWorkerFlow";
+    private static final String START_METHOD = "execution(* alluxio.worker.AlluxioWorker.main(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
```

Comment:
Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.